exec/java-exec/src/main/java/org/apache/drill/exec/store/easy/text/compliant/TextParsingSettings.java

```diff
@@ -57,6 +56,14 @@ public void set(TextFormatConfig config){
     this.delimiter = bSafe(config.getFieldDelimiter(), "fieldDelimiter");
     this.comment = bSafe(config.getComment(), "comment");
     this.skipFirstLine = config.isSkipFirstLine();
+    //TODO: avoid specific extension based check
```

Comment:
csvh is part of the format plugin config
added extractHeader property and cleared the TextParsingSettings code. will upload with other changes.